docs/manual/cli/user.en.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "contao:user"
+description: "Create and list Contao users."
+aliases:
+    - /en/cli/user/
+---
+
+
+After the [database](/en/cli/migrate/) has been created, Contao [users](/en/user-management/users/) can be created here. You can get 
+an overview of the commands via:
+
+```bash
+php vendor/bin/contao-console contao:user --help
+```
+
+## contao:user:list
+
+Lists existing Contao users:
+
+| Option | Beschreibung |
+| --- | --- |
+| `--help`   | Options overview |
+| `--admins` | Show administrators only |
+
+```bash
+php vendor/bin/contao-console contao:user:list --help
+```
+
+
+## contao:user:create
+
+Creates a Contao user:
+
+| Option | Beschreibung |
+| --- | --- |
+| `--help`   | Options overview |
+| `--username` | User name |
+| `--name` | Full user name |
+| `--email` | The user e-mail address |
+| `--password` | User password |
+| `--admin` | User is administrator |
+| `--group` | Groups to be assigned to the user |
```

Comment:
Would be good to explain that you could use:
```contao:user:create --group=1 --group=3 --group=5```
to assign multiple groups at once as well